plugins/typescript-semantics.js

```diff
@@ -23,6 +23,8 @@ module.exports = {
 
     // disallows awaiting a value that is not a `Thenable`
     '@typescript-eslint/await-thenable': 2,
+    // suggest `NonNullable<unknown>` as an alternative for `{}` and `Object`
+    '@typescript-eslint/ban-types': 1,
```

Comment:
This (non-semantic) rule is already covered in `./typescript.ts`.

Reply:
It has been removed!